client/coral-admin/src/routes/Moderation/graphql.js

```diff
@@ -124,12 +128,16 @@ function determineLatestChange(comment) {
 }
 
 function reconstructPreviousCommentState(comment) {
-  const history = comment.status_history;
+  const statusHistory = comment.status_history;
+  const bodyHistory = comment.body_history;
   const actions = comment.actions;
```

Comment:
Is there a reason why this style is used rather than destructuring?

```diff
- const statusHistory = comment.status_history;
- const bodyHistory = comment.body_history;
- const actions = comment.actions;
+ const {
+   status_history: statusHistory,
+   body_history: bodyHistory,
+   actions,
+ } = comment;
```